### PR TITLE
MySql deployment

### LINKED ISSUE #12: Automation mysql installation
Implement a bash script which install mysql on a fedora 30 VM

### PR DESCRIPTION
**Pull request by BarakMerdler**
fix #12

### CODE REVIEW

**Comment by BarakMerdler** [comment]
adding to bootstrap.sh script to run MySQL
Adding txt file for config .rpm
changed memory to 1024 to run MySQL
#12

File: Vagrantfile [hunk 1]
```diff
@@ -1,6 +1,9 @@
 Vagrant.configure("2") do |config|
   config.vm.box = "fedora/30-cloud-base"
-  config.vm.provision "shell",
-  path:"scripts/bootstrap.sh"
+  config.vm.provision "file", source: "scripts/start-sql-config.txt", destination: "/tmp/start-sql-config.txt"
```
Comment (lmilbaum) [inline Vagrantfile]: You do not need that. All files are synced to guest into '/vagrant' directory

File: scripts/bootstrap.sh [hunk 2]
```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env bash
-
+mv /tmp/start-sql-config.txt /etc/yum.repos.d/mysql-community.repo
```
Comment (lmilbaum) [inline scripts/bootstrap.sh]: Again, you don't need this 2 step action. The file - mysq-community.repo can be found in /vagrant during provisioning.

File: scripts/bootstrap.sh [hunk 3]
```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env bash
-
+mv /tmp/start-sql-config.txt /etc/yum.repos.d/mysql-community.repo
 dnf -y update
+dnf -y install mysql-community-server
```
Comment (lmilbaum) [inline scripts/bootstrap.sh]: You can "merge" this line with line 7 and move the systemctl commands after line 7.

**Review by lmilbaum — CHANGES_REQUESTED** [review]

**Comment by BarakMerdler** [comment]
@lioramilbaum commit changes after the last review.
delete the `config.vm.provision "file"` from vagrantfile
moved the mysq-community.repo to /vagrant
merged installation mysql with python3-pip

**Review by lmilbaum — APPROVED** [review]
LGTM

**Pull request merged**